### ISSUE TITLE
Switching from the admin user to a managed user cause Kodi to crash

### ISSUE THREAD

**Issue by keithah** [plexinc/plex-for-kodi#87]
Open Kodi.
Open Plex for Kodi addon from video addons.
Automatically signed in as admin user.
Switch to another managed user.
Kodi crashes.

[kodi_stacktrace-17.1-RC1 Git_20170223-1a80820-20170319-214159.txt](https://github.com/plexinc/plex-for-kodi/files/863525/kodi_stacktrace-17.1-RC1.Git_20170223-1a80820-20170319-214159.txt)

[kodi-31.txt](https://github.com/plexinc/plex-for-kodi/files/863527/kodi-31.txt)

ref: https://forums.plex.tv/discussion/264066/bug-report-logs-inside-switching-from-the-admin-user-to-a-managed-user-cause-kodi-to-crash#latest

**Comment by danjames92** [comment]
New log from latest testing repo version:

http://paste.ubuntu.com/24353458/

Issue seems to start at line 618 from when I switch user?

**Comment by ruuk** [comment]
@danjames92: There isn't a Kodi crash in your log. You seem to start the addon and then shut it down normally, and then start the Kodi Log Uploader where the log ends. What is the issue you are referring to?

**Comment by danjames92** [comment]
The log I provided covers the issue as far as I know. It crashes kodi itself. Not just the addon from what I can tell as you get a Kodi splash screen come up. Are you not able to reproduce the issue yourself then? Just switch from the admin user to a managed one and it crashes Kodi. When it autostart again, it's signed in as the managed user.

**Comment by ruuk** [comment]
Well, there is no crash in that log - you wouldn't have been able to have used Kodi Log Uploader if Kodi had crashed, and that is the last thing in the log (as is expected when you upload via that add-on). I only mention it to make sure we were talking about an actual Kodi crash. In any case this morning I got my Kodi to crash on my Linux dev machine after trying several time, then I updated to 17.1 stable and haven't been able to do it since (I switched users dozens of times with no issue). On the other hand, on my Shield (which is also running 17.1 stable) I was able to get Kodi to crash after a few attempts. Unfortunately other than possibly giving me a vague idea of when it's happening if I can get enough logs, the logs don't give a lot of useful information, so this is going to be difficult to get to the bottom of.
